libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.resolver.ts

```diff
@@ -44,4 +45,16 @@ export class NationalRegistryXRoadResolver {
       user.authorization,
     )
   }
+
+  @ResolveField('history', () => [NationalRegistryResidence])
```

Comment:
I believe you have a mismatch here... nestjs/graphql should flag this.
The field is called `residenceHistory`, not `history`